src/discussions/comments/comment-icons/CommentIcons.jsx

```diff
@@ -0,0 +1,46 @@
+import { faFlag, faStar as faEmptyStar } from '@fortawesome/free-regular-svg-icons';
+import { faEllipsisV, faStar as faSolidStar } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Button from '@edx/paragon/dist/Button';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function CommentIcons({ abuseFlagged, following }) {
+  return (
+    <div className="d-flex flex-column icons">
+      {/* Only show the star if the comment has a following attribute, indicating it can be followed */}
+      { following !== undefined && (
+        following
+          ? (
+            <Button variant="link" className="p-0" size="xs">
+              <FontAwesomeIcon icon={faSolidStar} />
+            </Button>
+          ) : (
+            <Button variant="link" className="p-0" size="xs">
+              <FontAwesomeIcon icon={faEmptyStar} />
+            </Button>
+          )
+      )}
+      { abuseFlagged && (
+        <Button variant="link" className="p-0" size="xs">
+          <FontAwesomeIcon icon={faFlag} />
+        </Button>
+      ) }
+      <Button variant="link" className="p-0" size="xs">
+        <FontAwesomeIcon icon={faEllipsisV} />
+      </Button>
```

Comment:
Not all the icons we need are available on paragon yet. Instead of partially replacing them and ending up with an inconsistent UI and potential issues due to different sizing, perhaps we can leave this for a future PR when all the icons are available? 
Does that sound OK?

Reply:
sounds good to me.